CMakeLists.txt

```diff
@@ -87,6 +87,20 @@ find_package(MPI REQUIRED)
 
 find_package(FFTW3 REQUIRED COMPONENTS DOUBLE MPI)
 
+if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+    set(FANS_LIB_DEFAULT OFF)
+endif ()
```

Comment:
With this construction, the option will always default to `OFF`, regardless of the project being top-level or not. That's because if not, the variable will not be defined and thus the option command will effectively not be provided a default value, which invokes the implicit default `OFF`. So this could be omitted I think.

Reply:
I want the compilation of the Python library to be `OFF` by default. How can we ensure that?

Reply:
These lines are only necessary if the default value should depend on wether the project is built top level or as an included dependency in another project. Thus we can just delete them and change the following `option()` call to
```CMake
option(FANS_LIB "Building FANS as a library to be used by the Micro Manager." OFF)
```